extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxKeycloakRecorder.java

```diff
@@ -25,7 +22,14 @@ public void setup(OidcConfig config, RuntimeValue<Vertx> vertx, BeanContainer be
         // Base IDP server URL
         options.setSite(config.authServerUrl);
         // RFC7662 introspection service address
-        options.setIntrospectionPath(config.introspectionPath);
+        if (config.introspectionPath.isPresent()) {
+            options.setIntrospectionPath(config.introspectionPath.get());
+        }
+
+        // RFC7662 JWKS service address
+        if (config.jwksPath.isPresent()) {
+            options.setIntrospectionPath(config.jwksPath.get());
```

Comment:
This looks like a copy/paste error?

Reply:
It does, thanks for catching it

Reply:
Funny, I was staring at the code yesterday evening for a long time :-)